lasagne/layers/recurrent.py

```diff
@@ -232,6 +256,10 @@ def step_masked(input_dot_W_n, mask_n, hid_previous, *args):
             hid_init = T.dot(T.ones((num_batch, 1)), self.hid_init)
 
         non_seqs = helper.get_all_params(self.hidden_to_hidden)
+        num_params_hid_to_hid = len(non_seqs)
```

Comment:
Is it possible to move this above the definition of `step`?  I'd prefer to have this variable defined before it's mentioned in the code (on line 218)